README.md

```diff
@@ -43,8 +43,9 @@ Breaking Changes!
 
 ## Usage
 
-* One can visit `/_ping.php` to get a status
-* By using `?debug=hash` query, one can get status check status table, and time profiling table. The `hash` is 4 first letters of the salt in `settings.php`.
+* Visit `/_ping.php` to get a system status
```

Comment:
Actually, in `settings.php` is also one reference here too to "hash":
```
// For debugging add "?debug=test" to the query - 4 letters of the hash.
```
Should be:
```
// In certain situations `hash_salt` is used to generate the debug token.
```